Oqtane.Client/Resources/Modules/Controls/AuditInfo.resx

```diff
@@ -132,4 +132,7 @@
   <data name="Deleted" xml:space="preserve">
     <value>Deleted</value>
   </data>
+  <data name="DateTimeFormat" xml:space="preserve">
+    <value>MMM dd yyyy HH:mm:ss</value>
```

Comment:
The date format might different from culture to another

Reply:
I know, but right now it is hardcoded in the framework, so thats why i included it in the localization.